src/pyfmi/exceptions.py

```diff
@@ -23,6 +23,12 @@ class FMUException(Exception):
     """
     pass
 
+class FMIModel_Exception(Exception):
```

Comment:
Can we create FMIModelException and extend from this one? Not sure if we can raise a warning for the one with `_` to flag for future removal.

Reply:
Added, also raising the new one; will not affect anyone doing except `FMUModel_Exception`.

I propose we get rid of the old one with PyFMI v3.

Reply:
Great!